library/src/app/components/account/account-list/account-list.component.html

```diff
@@ -1,121 +1,106 @@
 <ng-container *ngIf="(isRecoverable$ | async) !== false; else error">
+  <app-navigation [routingData]="routingData"></app-navigation>
   <ng-container *ngIf="(isLoading$ | async) !== true; else loading">
     <ng-container *ngIf="configService.getConfig$() | async as config">
-      <app-navigation [routingData]="routingData"></app-navigation>
-      <ng-container *ngIf="accountData$ | async as data">
-        <div id="account-value" class="cybrid-balance">
-          <span class="mat-hint">{{ 'accountList.total' | translate }}</span>
+      <ng-container *ngIf="totalAccountsValue$ | async as value">
+        <div id="account-value" class="cybrid-total-value">
+          <p class="mat-hint">{{ 'accountList.total' | translate }}</p>
           <div>
             <h1>
-              {{
-                currentFiat.symbol +
-                  (data.balance | currency : '' : '' : '' : config.locale)
-              }}
-              <span class="mat-hint">{{ currentFiat.code }}</span>
+              {{ value | assetFormat : config.fiat }}
             </h1>
           </div>
         </div>
-        <app-account-balance [account]="data.fiatAccount"></app-account-balance>
       </ng-container>
-      <table
-        id="assetList"
-        mat-table
-        matSort
-        [dataSource]="dataSource"
-        class="mat-button-toggle-group-appearance-standard"
-        [ngClass]="{ 'cybrid-paginator': dataSource.data.length > 5 }"
-      >
-        <ng-container matColumnDef="asset">
-          <th mat-header-cell *matHeaderCellDef mat-sort-header="">
-            <div class="cybrid-sort-header">
-              <span>{{ 'accountList.account.header' | translate }}</span>
-              <span class="mat-hint cybrid-h5 cybrid-code">{{
-                'accountList.account.subheader' | translate
-              }}</span>
-            </div>
-          </th>
-          <td id="asset" mat-cell *matCellDef="let asset">
-            <div class="cybrid-asset-cell">
-              <img
-                class="cybrid-icon"
-                src="{{ asset.asset.url }}"
-                alt="Crypto currency icon"
-              />
-              <div class="cybrid-column">
-                <div class="cybrid-currency-wrapper">
+
+      <div class="cybrid-table">
+        <mat-progress-bar *ngIf="isLoadingResults$ | async" mode="indeterminate"></mat-progress-bar>
+        <table id="accountList" class="mat-button-toggle-group-appearance-standard"
+          [ngClass]="{ 'cybrid-paginator': totalRows > 5 }" mat-table [dataSource]="dataSource" matSort
```

Comment:
Shouldn't this 5 be an environment variable?

Reply:
It's declared in the component. I've referenced this variable in the template now.